doc/prometheus-metric-schedule.md

```diff
@@ -82,9 +82,9 @@ JDBC Statement schema describes key metrics of JDBC SQL Statement, which include
 | application_jdbc_statement_p999   | double  | TP99.9: The JDBC method execution duration in milliseconds for 99.9% user.                            |
 
 #### Dedicated label
-| Label Name | Essential | Description                     |
-|:-----------|:---------:|:--------------------------------|
-| signature  |   true    | Executed JDBC method signature. |
+| Label Name | Essential |           Value Example            | Description                                                                                                                                                                                                                                         |
+|:-----------|:---------:|:----------------------------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| signature  |   true    | "440580e6c3215eceb4ef992d60adde9a" | Executed JDBC method signature. By default, It is md5 of sql like "select * from data". It can be restored to sql by turning off the switch: `plugin.observability.jdbc.sql.compress.enabled=false`. [Details](./user-manual.md#tracing-and-metric) |
```

Comment:
```suggestion
| signature  |   true    | "440580e6c3215eceb4ef992d60adde9a" | Executed JDBC method signature. By default, It is an MD5 of SQL like "select * from data". It can be SQL statement via turning off the switch: `plugin.observability.jdbc.sql.compress.enabled=false`. [Details](./user-manual.md#tracing-and-metric) |
```